src/traceViewer/web/index.tsx

```diff
@@ -39,7 +40,29 @@ function platformName(): string {
   document!.defaultView!.addEventListener('blur', event => {
     document.body.classList.add('inactive');
   }, false);
+
   document.documentElement.classList.add(platformName());
+
+  monaco.editor.setTheme('vs-light');
+  const monacoEnvironment: monaco.Environment = {
+    getWorkerUrl(moduleId, label) {
+      switch (label) {
+        case 'json':
+          return './monaco.json.worker.bundle.js';
+        case 'css':
+          return './monaco.css.worker.bundle.js';
+        case 'html':
+          return './monaco.html.worker.bundle.js';
+        case 'typescript':
+        case 'javascript':
+          return './monaco.ts.worker.bundle.js';
+        default:
+          return './monaco.editor.worker.bundle.js';
+      }
+    }
+  };
+  (self as any).MonacoEnvironment = monacoEnvironment;
```

Comment:
I thought you didn't need to do this anymore. Maybe old monaco? https://github.com/microsoft/monaco-editor/issues/1795

Reply:
I see the global types defined, but for some reason ts still complains.